frontend/src/metabase/entities/data-apps/data-apps.js

```diff
@@ -0,0 +1,50 @@
+import { color } from "metabase/lib/colors";
+import { createEntity } from "metabase/lib/entities";
+
+import { DataAppSchema } from "metabase/schema";
+import { CollectionsApi, DataAppsApi } from "metabase/services";
+
+import { DEFAULT_COLLECTION_COLOR_ALIAS } from "../collections/constants";
+
+import { createForm } from "./forms";
+import { getDataAppIcon } from "./utils";
+
+const DataApps = createEntity({
+  name: "dataApps",
+  nameOne: "dataApp",
+
+  displayNameOne: "app",
+  displayNameMany: "apps",
+
+  path: "/api/app",
+  schema: DataAppSchema,
+
+  api: {
+    create: async ({ name, description, ...dataAppProps }) => {
+      const collection = await CollectionsApi.create({
+        name,
+        description: description || null,
+        parent_id: null, // apps should always live in root collection
```

Comment:
with move functionality enabled, they can actually get moved out of the root collection

Reply:
Yeah, this is something we should remove soon. Think it's going to disappear from the FE once we add a dedicated "app landing page" (i.e. what you see when you click an app in the main sidebar). And I've added a task to block "moves" on the BE for app collections to the epic #24861